azure-pipelines.yml

```diff
@@ -89,7 +89,21 @@ jobs:
   - template: /eng/performance/benchmark_jobs.yml
     parameters:
       osName: windows
-      osVersion: 10
+      osVersion: RS5
+      kind: micro
+      architecture: x64
+      pool: Hosted VS2017
+      queue: Windows.10.Amd64.ClientRS5.Perf # using a dedicated private Helix queue (perfsnakes)
+      csproj: src\benchmarks\micro\MicroBenchmarks.csproj
+      runCategories: 'coreclr corefx'
+      benchviewCategory: 'coreclr'
+      frameworks: # for private jobs we want to benchmark .NET Core 3.0 only
+        - netcoreapp3.0
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'Schedule')) }}:
```

Comment:
Why the Windows version depends on the trigger (Windows RS**5** for PullRequest and RS**1** for the Scheduled triggers)? How are we going to differentiate the uploaded results in the Reporting System (there can be some differences I guess)?

```yml
# Windows x64 micro benchmarks, private job
- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
  - template: /eng/performance/benchmark_jobs.yml
    parameters:
      osVersion: RS5
      queue: Windows.10.Amd64.ClientRS5.Perf # using a dedicated private Helix queue (perfsnakes)
- ${{ if and(ne(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'Schedule')) }}:
  - template: /eng/performance/benchmark_jobs.yml
    parameters:
      osVersion: RS1
      queue: Windows.10.Amd64.ClientRS1.Perf # using a dedicated private Helix queue (perfsnakes)
```

Reply:
To question 1: The RS5 queue is the new queue for the PerfTigers, the RS1 queue is the PerfSnakes. We want to run on the PerfTigers for all runs, but for a little while, we also want to maintain some duplicate runs on the snakes so we can understand if we're seeing a regression compared to earlier runs, up until the point that we have enough history from the Tigers.

In the new database, we retain information about the queue that a job was run in, so we can separate the results out based on queue. We did this knowing we would have instances where we changed machines, and wanted to keep old results separate from new results.

Reply:
@adiaaida thanks for the explanation! now I get it